From 389fe618467bd16f910a4e79725e72997c496180 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 16 Jul 2020 14:16:59 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10.dtsi export wireless-wlan node

Since wireless-wlan node uses gpio as:
WIFI,host_wake_irq = <&gpio0 RK_PB0 GPIO_ACTIVE_HIGH>;

It should allow to disable the node.

Change-Id: I27cc8acbb84025ae5f8e76e4c3e7aee36907729e
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index ae06aa1278c2..3d4f849fd048 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -195,7 +195,7 @@
 		status = "okay";
 	};
 
-	wireless-wlan {
+	wireless_wlan: wireless-wlan {
 		compatible = "wlan-platdata";
 		rockchip,grf = <&grf>;
 		clocks = <&rk809 1>;
-- 
2.35.3

